helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -243,6 +243,17 @@ receivers:
     listenAddress: 0.0.0.0:9080
   {{- end }}
 
+  {{- if .Values.targetallocator.enabled  }}
+  prometheus/crd:
```

Comment:
`cdr` is a bit confusing here. Why not `prometheus/ta`?

Reply:
Sure